Documentation/tuning.md

```diff
@@ -0,0 +1,73 @@
+
+We recommend starting with our [recommended-resources-limit.yaml][recommended-resource-limits] example configuration, and adjusting it based on monitoring data over the period of a few weeks. It's much easier to start with higher values and lower them until you see pods restarting or when monitoring shows pods reaching their resource limits.
+
+## General Advice
+
+Below is some generally useful information to know when trying to tune the various components and understand _why_ we suggest specific values in our [recommended-resources-limit.yaml][recommended-resource-limits] example configuration.
+
+### reporting-operator
+
+The reporting-operator's most resource intensive task is querying Prometheus for metrics, storing the results in memory, converting them into a SQL insert statement, and batching those inserts to Presto.
+Performing imports is mostly a memory and IO intensive task, and thus increasing memory beyond defaults can help when dealing with a lot of metrics on larger or highly active clusters.
+
+### Presto
+
+Presto is the main component that does real computation intensive work.
+Additionally, it processes everything in memory when a query comes in, meaning it's primary resource constraint will often be memory, and CPU can be a problem when there's a lot of garbage collections occurring while processing a query.
+
+Presto itself doesn't store the data, and instead it accesses the data on demand when a query is executed.
+This means network is often a bottleneck, and can be something you may need to consider.
+
+When inserting data directly through Presto, this action is handled by the single Presto coordinator.
+This means throughput of data collection is bottlenecked by the coordinator's ability to write these results to storage.
+Reports on the other hand work on data already in storage, meaning Presto will split the work up across all available Presto pods.
+
+### Hive
+
+Hive metastore is often fairly idle, and is queried everytime Presto must interact with a table.
+These interactions are all fairly light weight, and thus the amount of resources the metastore requires isn't very large.
+However, it does retain quite a bit of information in memory, and can require more memory than the default values provide when it's been running for a longer period of time.
+
+The Hive server component is very lightly used and is only interacted with when creating new tables, or modifying partitions of an AWS Billing ReportDataSource table, and thus requires fewer resources.
+
+### HDFS
+
+By default, Metering installs HDFS for storage. While the amount of data isn't large in most cases, you do want to consider running multiple HDFS datanode replicas for redundancy.
+Because HDFS is commonly accessed by Presto when data is being stored and queried, it's also typically a component which more resources over time.
```

Comment:
I think this sentence is missing a verb...should it be `typically a component which uses more resources over time`?